pkgs/by-name/li/libbgcode/package.nix

```diff
@@ -36,5 +36,6 @@ stdenv.mkDerivation {
     description = "Prusa Block & Binary G-code reader / writer / converter";
     license = licenses.agpl3Only;
     maintainers = with maintainers; [ lach ];
+    platforms = platforms.unix;
```

Comment:
Why included in this PR?

Reply:
😕 Where did chat come from? I must have accidentally picked this when rebasing.